### PR TITLE
home screen

### PR DESCRIPTION
**Pull request by benedictdube**

### CODE REVIEW

File: client/pages/likedTrains.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta charset="UTF-8">
+  <title>Liked Trains | Train Tinder</title>
+  <link rel="icon" href="../resources/images/train_tinder_icon.png">
+  <link rel="stylesheet" href="../resources/style/app.css" />
+</head>
+<body>
+  <main>
+    <main id="liked-trains-page">
```
Comment (StevenPintoVC) [inline client/pages/likedTrains.html]: Main inside a main?

File: client/pages/likedTrains.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <header>
+        <button id="back-button" onclick="history.back()">Back</button>
+        <h1>Trains You've liked</h1>
+        <button id="settings-button" onclick="history.back()">
+          <i class="fa fa-cog"></i>
+        </button>
+      </header>
+      <span id="liked-profile">
```
Comment (StevenPintoVC) [inline client/pages/likedTrains.html]: Rather use article instead of span or div

File: client/pages/home.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<head>
+  <meta charset="UTF-8">
+  <title>Home | Train Tinder</title>
+  <link rel="icon" href="../resources/images/train_tinder_icon.png">
+  <link rel="stylesheet" href="../resources/style/app.css" />
+  <link rel="stylesheet" type="text/css"
+        href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css" />
+</head>
+<body>
+  <section id="home-content">
```
Comment (StevenPintoVC) [inline client/pages/home.html]: This should be main

File: client/pages/home.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <h1>Train Tinder</h1>
+    <span id="home-options">
```
Comment (StevenPintoVC) [inline client/pages/home.html]: This should be a section

File: client/resources/style/app.css [hunk 5]
```diff
@@ -141,6 +147,158 @@ body{
...
+#home-content > #home-options > a > p {
+    font-size: var(--medium);
+    padding: var(--medium) ;
+}
+
+#liked-trains-page {
+    height: 100vh;
+    width: 100vw;
+    max-height: 100vh;
+    max-width: 100vw;
```
Comment (StevenPintoVC) [inline client/resources/style/app.css]: You can remove the max width and heights

File: client/pages/likedTrains.html (hunk 2 above)
Comment (StevenPintoVC) [reply client/pages/likedTrains.html]: Or section

File: server/src/main/resources/application.properties [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
-spring.datasource.username=admin
-spring.datasource.password=TrainTinder
+spring.datasource.url=${DB_URL}
+spring.datasource.username=${DB_USER}
+spring.datasource.password=${DB_PASS}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 
-spring.security.oauth2.resourceserver.jwt.issuer-uri=https://cognito-idp.eu-west-1.amazonaws.com/eu-west-1_BZAu5yKxc
\ No newline at end of file
+spring.security.oauth2.resourceserver.jwt.issuer-uri=${COGNITO}
```
Comment (StevenPintoVC) [inline server/src/main/resources/application.properties]: COGNITO_RESOURCE

**Review by StevenPintoVC — APPROVED** [review]

**Review by matt-Kretschmer — APPROVED** [review]

**Review by matt-Kretschmer — APPROVED** [review]

**Pull request merged**